File: (7 kyu) Exes and Ohs/7 kyu Exes and Ohs.js
// #1
// function XO(str, equal = 0) {
//     str.split``.forEach(c => {
//       'x' === c.toLowerCase() && equal++
//       'o' === c.toLowerCase() && equal--
//     })
//     return equal === 0
// }
// #2
// const XO = (str) =>
//   str.split``.reduce((a, c) => {
//     if ('x' === c.toLowerCase()) {
//       return a + 1;
//     } else if ('o' === c.toLowerCase()) {
//       return a - 1;
//     } else return a;
//   }, 0) === 0;
// #3
const XO = (str) =>
  str
    .toLowerCase()
    .split('')
    .reduce((a, c) => ('x' === c ? ++a : 'o' === c ? --a : a), 0) === 0;

File: (7 kyu) Exes and Ohs/7 kyu Exes and Ohs.py
# #1
# def xo(s):
#     equal = 0
#     for c in s:
#         if c.lower() == 'x': equal += 1
#         if c.lower() == 'o': equal -= 1
#     return equal == 0
#2
def xo(s):
    return s.lower().count('x') == s.lower().count('o')
